App/src/services/course.ts

```diff
@@ -17,3 +18,7 @@ export function getAllCourseSettings(): Promise<GetAllCourseSettingsResponse> {
 export function addCourse(course: Course): Promise<unknown> {
     return axios.post(CourseAPIEndpoints.AddCourse, course);
 }
+
+export function deleteCourse(courseDeletionRequest): Promise<unknown> {
```

Comment:
Can you create an interface for the courseDeletionRequest parameter, like we do in the other service files?

Reply:
Yes, done.